plugins/default/lib/disk_usage.py

```diff
@@ -44,5 +107,4 @@ def main():
     except:
         sys.exit(nagios.UNKNOWN)
```

Comment:
@jessesarn comparing this with the component health check, I realized we're not exiting with the proper exit codes, it's either 0 or nagios.UNKNOWN.

I'll send a PR to address it.